near-sdk-macros/src/lib.rs

```diff
@@ -43,6 +43,32 @@ struct NearMacroArgs {
     inside_nearsdk: Option<bool>,
 }
 
+fn has_nested_near_macros(item: TokenStream) -> bool {
+    syn::parse::<syn::Item>(item)
+        .ok()
+        .and_then(|item_ast| {
+            let attrs = match item_ast {
+                syn::Item::Struct(s) => s.attrs,
+                syn::Item::Enum(e) => e.attrs,
+                syn::Item::Impl(i) => i.attrs,
+                _ => vec![], // Other cases don't support near macros anyway
+            };
+
+            attrs.into_iter().find(|attr| {
+                let path_str = attr.path().to_token_stream().to_string();
+                path_str == "near" || path_str == "near_bindgen"
```

Comment:
this could be extracted to constant:
```rust
const NEAR_BINDGEN: &str = "near_bindgen";

pub(crate) fn near_bindgen_ident() -> syn::Ident {
    syn::Ident::new(NEAR_BINDGEN, Span::call_site())
}
```
and then
```rust
        let near_bindgen_ident = near_bindgen_ident();
        if let Some(metadata) = near_macro_args.contract_metadata {
            expanded = quote! {#[#near_sdk_crate::#near_bindgen_ident(#metadata)]}
        } else {
            expanded = quote! {#[#near_sdk_crate::#near_bindgen_ident]}
        }
```
done in another spot to have syntactic link via usages/references, but this remains unclear, if this is a good idea

Reply:
I would keep it as it is for sake of simplicity

Reply:
nope, this definetely should be done)). just not in the scope of an unrelated pr

Reply:
Not sure how it helps, I didn't get it :)